IORawData/SiPixelInputSources/src/PixelSLinkDataInputSource.cc

```diff
@@ -260,7 +260,7 @@ bool PixelSLinkDataInputSource::setRunAndEventInfo(edm::EventID &id,
   buffers = std::make_unique<FEDRawDataCollection>();
 
   //  uint32_t currenteventnumber = (m_data >> 32)&0x00ffffff;
```

Comment:
This line can be removed, because no `currenteventnumber` variable exists anywhere in this code
```suggestion
```